config/application.rb

```diff
@@ -8,7 +8,19 @@
 
 module FlashCardManager
   class Application < Rails::Application
-    # Settings in config/environments/* take precedence over those specified here.
+    #AWS settings
+    Rails.application.configure do
+      config.paperclip_defaults = {
+        storage: :s3,
+        s3_credentials: {
+          bucket: ENV["AWS_BUCKET"],
+          s3_host_name: ENV["AWS_HOSTNAME"],
+          access_key_id: ENV["AWS_ACCESS_KEY_ID"],
+          secret_access_key: ENV["AWS_SECRET_ACCESS_KEY"]
+          }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.